.github/workflows/docker-health-check.yml

```
@@ -0,0 +1,70 @@
+          echo "✅ AI 서비스가 정상 작동 중입니다."
+        else
+          echo "❌ AI 서비스 응답 실패: HTTP $response"
+          exit 1
+        fi
+        
+    - name: Spring Backend 연결 테스트
+      run: |
+        python -c "
+        import requests
+        import os
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)